packager/launchPackager.command

```diff
@@ -16,5 +16,4 @@ pushd "$THIS_DIR"
 source packager.sh
 popd
 
-echo "Process terminated. Press <enter> to close the window"
-read
```

Comment:
I believe the purpose of this statement was to keep the `xinput` window open.  By removing this, I believe you'll break that.  `read` essentially waits for the first input received from `stdin` until `LFS` has been supplied, then it exits.